odd_collector_azure/adapters/azure_sql/mappers/metadata.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+
+_METADATA_SCHEMA_URL_PREFIX: str = (
+    "https://raw.githubusercontent.com/opendatadiscovery/opendatadiscovery-specification/main/specification/"
+    "extensions/mysql.json#/definitions/Mysql"
+)
+
+_data_set_metadata_schema_url: str = _METADATA_SCHEMA_URL_PREFIX + "DataSetExtension"
+_data_set_field_metadata_schema_url: str = (
+    _METADATA_SCHEMA_URL_PREFIX + "DataSetFieldExtension"
+)
+
+
+def convert_bytes_to_str(value: Optional[bytes]) -> Optional[str]:
```

Comment:
`odd_collector.helpers` package has same function, reuse it

Reply:
Created the helpers package as on odd-collector.